### DISCUSSION TITLE
Getting Sciprt Error While using the Custom WebPart in the SharePoint Site

### DISCUSSION THREAD

**Discussion by sowmiyavigneshkumar** [pnp/sp-dev-fx-webparts#3639] [Q&A] [answered]
I'm Using the [react-rss-reader]. The webpart is running successfully in the workbench. But, When I deploy the solution and use them in the SharePoint site I'm getting the below error

Something went wrong
If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS
ERROR:
Script error for: f489b1fd-98bf-4b41-8db0-85d5018ba484_1.0.0/RssReaderWebPartStrings
http://requirejs.org/docs/errors.html#scripterror

CALL STACK:
Error: Script error for: f489b1fd-98bf-4b41-8db0-85d5018ba484_1.0.0/RssReaderWebPartStrings
http://requirejs.org/docs/errors.html#scripterror
    at B (eval at e (https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_892a9e40037b2b7c66427e01fa58ca3a.js:70:746346), <anonymous>:8:252)
    at HTMLScriptElement.onScriptError (eval at e (https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_892a9e40037b2b7c66427e01fa58ca3a.js:70:746346), <anonymous>:30:244)

**Comment by Fabius74** [comment]
I solved this problem by clearing the browser cache (Chrome in my case)

**Reply by sowmiyavigneshkumar** [reply]
it didn't work in my case. Thanks

**Comment by sowmiyavigneshkumar** [comment, marked as answer]
I got this fixed by enabling the CDN for the SharePoint Site.
